docs/Company/DNA.mdx

```diff
@@ -0,0 +1,34 @@
+---
+sidebar_position: 2.3
+sidebar_label: "DNA"
+---
+
+import MaintainerTag from "@site/src/components/MaintainerTag";
+import ControlledDocBanner from "@site/src/components/ControlledDocBanner";
+
+<MaintainerTag maintainerEmails={["akshat.mittal@openenergytransition.org", "fabrizio.finozzi@openenergytransition.org"]} />
+
+# DNA & Core Values
+
+<ControlledDocBanner />
+
+As our team continues to grow, it’s essential to align on the core values and principles that define us—our DNA. This foundation sets us apart and drives our mission. Below is a curated list of resources that embody these principles and should be on everyone’s radar.
+
+## Free and Open-Source Software
+<a href="https://www.youtube.com/watch?v=k0RYQVkQmWU">Revolution OS -1.5h video</a><br>
+<a href="https://www.gnu.org/proprietary/proprietary.html">Free Software Foundation. Proprietary Software Is Often Malware</a><br>
+<a href="https://www.linkedin.com/posts/tom-brown-226b191a0_freethemodels-activity-7265653451135635456-xkQT/">Brown 2024. Leading black-box energy planning tool embargoes Technical University of Berlin.</a>
+
+## Open Energy Modelling
+<a href="https://www.nature.com/articles/542393a">Energy scientists must show their workings.</a><br>
+<a href="https://openmod-initiative.org/manifesto.html">Open energy modelling manifesto</a><br>
+<a href="https://www.sciencedirect.com/science/article/pii/S2211467X17300809">Opening the black box of energy modelling: Strategies and lessons learned</a><br>
+<a href="https://www.sciencedirect.com/science/article/abs/pii/S1364032118307743">Are open source energy system optimization tools mature enough for serious use?</a>
+
+## Open data
+<a href="https://forum.openmod.org/t/open-letter-calling-for-cc-by-4-0-licenses-on-structured-information-describing-energy-systems/3578">Morrison & Parzen 2023. Open energy system modelers call for CC‑BY‑4.0 data licensing wherever possible — our world needs comprehensive usable and reusable data to transition to net‑zero.</a>
+
+## Open source optimization solver
+<a href="https://zenodo.org/records/12703174">Parzen 2023. Optimization solvers: the missing link for a fully open- source energy system modelling ecosystem.</a>
+<br><br>
```

Comment:
- <br> tag to - <br /> because its self closing
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#examples